package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@carto/carto-vl",
   "version": "0.3.0",
+  "sideEffects": false,
```

Comment:
what is this?

Reply:
This marks the code as pure, to enable tree shaking, see https://webpack.js.org/guides/tree-shaking/